Content.Shared/Roles/JobRequirements.cs

```diff
@@ -126,6 +126,20 @@ public static class JobRequirements
                     }
 
                     var deptDiff = deptRequirement.Time.TotalMinutes - playtime.TotalMinutes;
+                    var nameDepartment = Loc.GetString("role-timer-department-unknown");
+
+                    if (!prototypes.TryIndex<DepartmentPrototype>(deptRequirement.Department, out var departmentName))
+                    {
```

Comment:
It's generally best to avoid throwing exceptions in Content code. Logging an error is usually preferred.

Reply:
> It's generally best to avoid throwing exceptions in Content code. Logging an error is usually preferred.

@Tayrtahn okay, then I'll quickly fix the error for simple logging now if that's better.

Reply:
> It's generally best to avoid throwing exceptions in Content code. Logging an error is usually preferred.

@Tayrtahn I did it! 😄

Reply:
instead use ProtoId and then prototypes.Index()

Reply:
> instead use ProtoId and then prototypes.Index()

@deltanedas I don't quite understand what you're talking about. I used the same approach as in line 114.

Reply:
replace string in DepartmentTimeRequirement with `ProtoId<DepartmentPrototype>`